db/seeders/distributions_seeder.rb

```diff
@@ -0,0 +1,46 @@
+class DistributionsSeeder
+  attr_accessor :organization
+
+  def self.seed(organization)
+    new(organization).seed
+  end
+
+  def initialize(organization)
+    @organization = organization
+  end
+
+  def seed
+    20.times.each do
+      storage_location = random_record_for_org(organization, StorageLocation)
+      stored_inventory_items_sample = storage_location.inventory_items.sample(20)
+
+      distribution = create_distribution(storage_location)
+
+      stored_inventory_items_sample.each do |stored_inventory_item|
```

Comment:
Not a big thing, but I think this could be simplified by eliminating the intermediate variable. The vertical position of the assignment implies logical connection with the line above it, and that took a little extra effort to mentally parse.

Reply:
👍